.github/workflows/probot-auto-cc.yml

```diff
@@ -0,0 +1,18 @@
+name: Probot
+
+on:
+  issues:
+    types:
+      - labeled
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  auto-cc:
+    if: ${{ github.repository_owner == 'PyTorchLightning' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: carmocca/probot@v1
```

Comment:
I need an admin to publish the action, until then, we can use my own

Reply:
have you asked for it? I do not see it anywhere in communication...